projects/plugins/boost/app/modules/cache/Boost_Cache.php

```diff
@@ -51,6 +51,19 @@ public function __construct( $storage = null ) {
 			'cookies' => $_COOKIE, // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
 			'get'     => $_GET,   // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.NonceVerification.Recommended
 		);
+
+		$this->init_actions();
+	}
+
+	/*
+	 * Initialize the actions for the cache.
+	 */
+	protected function init_actions() {
+		add_action( 'transition_post_status', array( $this, 'delete_on_post_transition' ), 10, 3 );
+		add_action( 'transition_comment_status', array( $this, 'delete_on_comment_transition' ), 10, 3 );
+		add_action( 'comment_post', array( $this, 'delete_on_comment_post' ), 10, 3 );
+		add_action( 'edit_comment', array( $this, 'delete_on_comment_edit' ), 10, 2 );
+		add_action( 'switch_theme', array( $this, 'delete_cache' ) );
```

Comment:
Do we need `save_post` as well?

Reply:
I don't think we'll need that. The `transition_post_status` action fires when a post is saved, even if the [status hasn't changed](https://developer.wordpress.org/reference/hooks/transition_post_status/#comment-2977).